PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SignInWindow.java

```diff
@@ -105,6 +111,16 @@ private void checkAccountAvailability() {
         });
     }
 
+    protected JPanel createSouthAdditionalPanel() {
```

Comment:
south -> bottom?

Reply:
When will we call this method? Shall we add the panel into the form and make it visible if need?

Reply:
its inherited from `DialogWrapper`